datalad_container/resources/procedures/cfg_containers.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""Procedure to configure dataset for containers
+"""
+
+import sys
+
+from datalad.distribution.dataset import require_dataset
+from datalad.support import path as op
+
+ds = require_dataset(
+    sys.argv[1],
+    check_installed=True,
+    purpose='Containers dataset configuration')
+
+# unless taken care of by the template already, each item in here
+# will get its own .gitattributes entry to keep it out of the annex
+# give relative path to dataset root (use platform notation)
+force_in_git = [
+    op.join('environments','**','*.json'),
+    op.join('environments','*','image','repositories'),
```

Comment:
I don't think we want it OS specific here, probably should just be posix, so
```suggestion
    'environments/**/*.json',
    'environments/*/image/repositories',
```